File: StarLight/Source/Compiletime/Composestar/Repository/LanguageModel/ConditionExpressions/True.java
package Composestar.Repository.LanguageModel.ConditionExpressions;

import Composestar.Repository.LanguageModel.ConditionExpressions.Visitor.*;

/**
 * Summary description for True.
 */
public class True extends ConditionExpression implements IVisitable 
{
	public True()
	{
		//
		// TODO: Add Constructor Logic here
		//
	}

	public void Accept(IVisitor visitor)
	{
		visitor.VisitTrue(this);  
	}
    
    public String toString(){
        return "true";
    }

}

File: StarLight/Source/StarLightEntities/WeaveSpec/ConditionExpressions/True.cs
#region Using directives
using System;
using System.Collections.Generic;
using System.Text;
using System.Xml;
using System.Xml.Serialization;
using Composestar.StarLight.Entities.WeaveSpec.ConditionExpressions.Visitor;
#endregion

namespace Composestar.StarLight.Entities.WeaveSpec.ConditionExpressions
{
    /// <summary>
    /// True value.
    /// </summary>
    [Serializable]
    [XmlRoot("True", Namespace = "Entities.TYM.DotNET.Composestar")]
    public class TrueCondition : ConditionExpression, IVisitable
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="T:True"/> class.
        /// </summary>
        public TrueCondition()
        {

        } // True()

        /// <summary>
        /// Accepts the specified visitor.
        /// </summary>
        /// <param name="visitor">The visitor.</param>
        public void Accept(IVisitor visitor)
        {
            if (visitor == null)
                throw new ArgumentNullException("visitor");

            visitor.VisitTrue(this);
        } // Accept(visitor)

    } // class True

}
